src/components/components/club-info/club-info.js

```diff
@@ -107,7 +102,7 @@ const EmailInfo = ({ email }) => (
 const EventInfo = ({ event }) => (
```

Comment:
Could you move the content of this one to the Event element? (no need for a wrapper that just adds 2 elements

Reply:
This is not as easy as it seems and it would add complexity since this is part of the club info component and logic, where we create a list with specific markup and structure to ensure consistency and re-usability.